qt/applications/workbench/workbench/plotting/plotscriptgenerator/__init__.py

```diff
@@ -120,6 +122,10 @@ def generate_script(fig, exclude_headers=False):
     cmds.extend(generate_workspace_retrieval_commands(fig) + [""])
     cmds.append("{}, {} = {}".format(FIG_VARIABLE, AXES_VARIABLE, generate_subplots_command(fig)))
     cmds.extend(plot_commands)
+    if blank_axes:
```

Comment:
Sorry if I'm not understanding this very well, but do we need to add the blank axes commands only after the other plot commands? I just wondered if we could potentially add the `.axis('off')` command for blank axes directly in the loop above at line 88, rather than looping through any blank axes again here?